### DISCUSSION TITLE
gtkwave problems

### DISCUSSION THREAD

**Discussion by s095339** [bol-edu/HLS-SOC-Discussions#90] [Q&A]
大家好，我在寫lab-fir的時候，使用Makefile編譯並且透過gtkwave來看訊號波形

我發現gtkwave並沒有顯示所有的訊號，有的訊號有顯示 有的沒有。
不知道怎麼解決
如圖
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/118b54ab-4248-46c1-b195-f27d16d7586c)

我之後也有加入新的訊號 但一樣沒顯示  都只能跑出43個訊號

**Comment by kevin861222** [comment]
請問你匡起來的這些信號有宣告在 input 和 output嗎？
另外，你是用iverilog編譯嗎？

**Reply by s095339** [reply]
這些沒有 但有的沒有在input output的有實現

我不確定我用甚麼 我是用這個makefile
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78518884/f0cd51e1-3e88-440d-895c-615c057df549)

**Comment by kevinjantw** [comment]
> 大家好，我在寫lab-fir的時候，使用Makefile編譯並且透過gtkwave來看訊號波形
> 
> 我發現gtkwave並沒有顯示所有的訊號，有的訊號有顯示 有的沒有。 不知道怎麼解決 如圖 ![image](https://user-images.githubusercontent.com/78518884/273536517-118b54ab-4248-46c1-b195-f27d16d7586c.png)
> 
> 我之後也有加入新的訊號 但一樣沒顯示 都只能跑出43個訊號

不確定你的驗證模擬過程是否正確, 你也可以先試試在Filter裡輸入訊號關鍵字找看看

**Comment by s095339** [comment]
@kevinjantw 
@kevin861222 
感謝回復，我這幾天有在別的電腦測過，我想可能我自己電腦的問題
謝謝
